pkgs/applications/terminal-emulators/mlterm/default.nix

```diff
@@ -73,6 +74,8 @@
   bidi = true;
   # Open Type layout support, (substituting glyphs with opentype fonts)
   otl = true;
+  # See comment above for enableTools.mlimgloader
+  image = enableTools.mlimgloader;
```

Comment:
So if I'm understanding this correctly, mlimgloader implies image and the other way around too?

In that case we should probably add an `==` assertion.